README.md

```diff
@@ -20,60 +20,20 @@ Learn more about [Remix Stacks](https://remix.run/stacks).
 
 Not a fan of bits of the stack? Fork it, change it, and use `npx create-remix --template your/repo`! Make it your own.
 
-## Fly Setup
-
-1. [Install Fly](https://fly.io/docs/getting-started/installing-flyctl/)
-
-2. Sign up and log in to Fly
-
-```sh
-fly auth signup
-```
-
-## The Database
-
-In development, it's better to use a local database, The easiest way to do this is using [Docker](https://www.docker.com/get-started). To start your postgres database, first make sure you have docker running, then run the following command:
-
-```sh
-docker-compose up
-```
-
-That may take a moment to start up as it needs to get the postgres image from the Docker registry, after it's done, you'll need to migrate your database. With the database ready to accept connections, open a new tab and run this:
-
-```sh
-npx prisma migrate deploy
-```
-
-When this finishes successfully, it will say:
-
-> All migrations have been successfully applied.
-
-If you'd prefer not to use Docker, you can also use Fly's Wireguard VPN to connect to a development database (or even your production database). You can find the instructions to set up Wireguard [here](https://fly.io/docs/reference/private-networking/#install-your-wireguard-app), and the instructions for creating a development database [here](https://fly.io/docs/reference/postgres/).
```

Comment:
> [...] we'll want to keep a few of the steps manual since they'll want to start the docker container themselves.

@kentcdodds So I guess we need to keep the whole `The Database` section, as running `npx prisma migrate deploy` will only work once we've run `docker-compose up` ourselves then.

> We should be able to run `npx prisma generate` and `npm run build` for them though which would be helpful.

`npx prisma generate` isn't added to the `indie-stack` either, should I add it there too?

Reply:
Yeah! Also, let's swap `npx prisma migrate deploy` for `npx prisma migrate dev` for the setup instructions so people get the seed script run automatically. Thanks!